models/location_helpers.go

```diff
@@ -239,11 +234,11 @@ func (c BackupLocationConfig) Validate() (bool, error) {
 		err = checkPMMClientLocationConfig(c.PMMClientConfig)
 	}
 
-	if configCount > 1 {
-		return false, status.Error(codes.InvalidArgument, "Only one config is allowed.")
+	if configCount != 1 {
```

Comment:
API description requires exactly one config to be present.

Reply:
But when u want change location, you don't have to specify config.

Reply:
I don't get, you mean when user wants to change location name or description?

API should be `send only fields that should be changed` or `send whole object`, but it was a mix of these two approaches. So user should have sent `name` and `description` each time, but configs only when he wants to change it.

Reply:
No, user can send empty `name` or `description`, then it's not updated.

And with this change, if user don't want change config, so he don't send anything. It will fail because of `configCount == 0`

Reply:
Hmm, it differs from how we handle IA entities. So as I see location `description` field is optional when you create record, but once you set description you are not able to remove it.

Reply:
Yes, we can try use wrappers for this, when it's new API. So we can check empty vs not sent.

Reply:
I slightly remember that this was discussed somewhere. It seems that different API parts have different behavior, which is not good.

Well, let's discuss it at some meeting. Please ignore this for now.